### DISCUSSION TITLE
v2.17.0-alpha

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#538] [General]
* Support setting static IP configuration. (#193)
* More properly support sharing system DNS with downstream users. In particular, you can now share your "Private DNS" (aka DNS over TLS) with your users too, which Android natively does not support. (#533)
* Support Android Auto Backup. (#535)
* You can grant "Display over other apps" permission to this app on Android 14+ so that you can start this app from quick settings tiles. Thank you Google for breaking shit.
* Refined design.
* Fixes and improved performance. (#533, #534)

P.S. Feel free to let me know if Android 15 breaks things.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.16.11...v2.17.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/'>v2.17.0-alpha</a>.</em>

**Comment by jomiejoshiro** [comment]
Hi that app very good!👍🏽 Can u build this with magisk module to ttl 64 (ipv4&ipv6)?

**Comment by huzesama** [comment]
Please add the "Add to Whitelist" option to the connected device page.

**Reply by jomiejoshiro** [reply]
Device name?

**Reply by huzesama** [reply]
yeah

**Reply by jomiejoshiro** [reply]
Xiaomi all POCO version.. i mean latest android
